### DISCUSSION TITLE
Download failed

### DISCUSSION THREAD

**Discussion by Hagakui90** [aclap-dev/video-downloadhelper#745] [Help and Q&A]
### Your question

{
  "version": "9.0.1.22",
  "target": "google",
  "channel": "stable",
  "page_url": "https://web.davinci.tv/video?s=01HW7S068K930771E8V42B0TYG&v=1_q12hbhem",
  "error": {
    "details": "SideDownload error: : [tls @ 000001c867c8ee40] error:0A000119:SSL routines::decryption failed or bad record mac\r\n[tls @ 000001c867c8ee40] Unknown error\r\n[tls @ 000001c867c34800] error:0A000119:SSL routines::decryption failed or bad record mac\r\n[tls @ 000001c867c34800] Unknown error\r\n[tls @ 000001c86a61d600] error:0A000119:SSL routines::decryption failed or bad record mac\r\n[tls @ 000001c86a61d600] Unknown error\r\n[tls @ 000001c86a61dfc0] error:0A000119:SSL routines::decryption failed or bad record mac\r\n[tls @ 000001c86a61dfc0] Unknown error\r\n[tls @ 000001c86a61e400] error:0A000119:SSL routines::decryption failed or bad record mac\r\n[tls @ 000001c86a61e400] Unknown error\r\n[tls @ 000001c86a61dbc0] error:0A000119:SSL routines::decryption failed or bad record mac\r\n[tls @ 000001c86a61dbc0] Unknown error\r\n[tls @ 000001c86a61dd80] error:0A000119:SSL routines::decryption failed or bad record mac\r\n[tls @ 000001c86a61dd80] Unknown error\r\n[tls @ 000001c86a61d600] error:0A000119:SSL routines::decryption failed or bad record mac\r\n[tls @ 000001c86a61d600] Unknown error\r\n[tls @ 000001c86a61dbc0] error:0A000119:SSL routines::decryption failed or bad record mac\r\n[tls @ 000001c86a61dbc0] Unknown error\r\n[tls @ 000001c86a61e300] error:0A000119:SSL routines::decryption failed or bad record mac\r\n[tls @ 000001c86a61e300] Unknown error\r\n[tls @ 000001c86a61da40] error:0A000119:SSL routines::decryption failed or bad record mac\r\n[tls @ 000001c86a61da40] Unknown error\r\n[tls @ 000001c86a61d880] error:0A000119:SSL routines::decryption failed or bad record mac\r\n[tls @ 000001c86a61d880] Unknown error\r\n[tls @ 000001c86a61e300] error:0A000119:SSL routines::decryption failed or bad record mac\r\n[tls @ 000001c86a61e300] Unknown error\r\n[tls @ 000001c86a61e400] error:0A00010B:SSL routines::wrong version number\r\n[tls @ 000001c86a61dfc0] error:0A000119:SSL routines::decryption failed or bad record mac\r\n[tls @ 000001c86a61dfc0] Unknown error\r\n[tls @ 000001c86a61e0c0] error:0A000119:SSL routines::decryption failed or bad record mac\r\n[tls @ 000001c86a61e0c0] Unknown error\r\n[tls @ 000001c86a61de40] error:0A000119:SSL routines::decryption failed or bad record mac\r\n[tls @ 000001c86a61de40] Unknown error\r\n[tls @ 000001c86a61e400] error:0A000119:SSL routines::decryption failed or bad record mac\r\n[tls @ 000001c86a61e400] Unknown error\r\n[tls @ 000001c86a61e180] error:0A000119:SSL routines::decryption failed or bad record mac\r\n[tls @ 000001c86a61e180] Unknown error\r\n[tls @ 000001c86a61d880] error:0A000119:SSL routines::decryption failed or bad record mac\r\n[tls @ 000001c86a61d880] Unknown error\r\n[tls @ 000001c86a61dd80] error:0A000119:SSL routines::decryption failed or bad record mac\r\n[tls @ 000001c86a61dd80] Unknown error\r\n[tls @ 000001c86a61d880] error:0A000119:SSL routines::decryption failed or bad record mac\r\n[tls @ 000001c86a61d880] Unknown error\r\n[tls @ 000001c86a61dd80] error:0A000119:SSL routines::decryption failed or bad record mac\r\n[tls @ 000001c86a61dd80] Unknown error\r\n[tls @ 000001c86a61da40] error:0A000119:SSL routines::decryption failed or bad record mac\r\n[tls @ 000001c86a61da40] Unknown error\r\n[tls @ 000001c86a61dbc0] error:0A000119:SSL routines::decryption failed or bad record mac\r\n[tls @ 000001c86a61dbc0] Unknown error\r\n[tls @ 000001c86a61dd80] error:0A000119:SSL routines::decryption failed or bad record mac\r\n[tls @ 000001c86a61dd80] Unknown error\r\n[tls @ 000001c86a61d600] error:0A000119:SSL routines::decryption failed or bad record mac\r\n[tls @ 000001c86a61d600] Unknown error\r\n[tls @ 000001c86a61dfc0] error:0A000119:SSL routines::decryption failed or bad record mac\r\n[tls @ 000001c86a61dfc0] Unknown error\r\n[tls @ 000001c86a61e240] error:0A000119:SSL routines::decryption failed or bad record mac\r\n[tls @ 000001c86a61e240] Unknown error\r\n[tls @ 000001c86a61e180] error:0A000119:SSL routines::decryption failed or bad record mac\r\n[tls @ 000001c86a61e180] Unknown error\r\n[aac_adtstoasc @ 000001c869930940] Error parsing ADTS frame header!\r\n[mp4 @ 000001c86a60ad40] Error applying bitstream filters to an output packet for stream #1: Invalid data found when processing input\r\n[aost#0:1/copy @ 000001c86a61c9c0] Error submitting a packet to the muxer: Invalid data found when processing input\r\n[out#0/mp4 @ 000001c86a613c40] Error muxing a packet\r\n",
    "downloadable_id": "downloadable_523484799267614",
    "error": "coapp_failure",
    "id": "coapp_failure_679a6edf-0044-4667-b721-deb400ab982b"
  }
}

### Addon & CoApp version and details

Download failed

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I'm just another user.

I went to the URL you provided in the page_url tag in your problem report.  It automatically redirected me to the site home page where it nagged me to buy a subscription.  That's not going to happen.  I can't investigate this.  I'm going to guess the content is protected by DRM, but that's just a guess.  Maybe you can confirm that.

**Reply by Hagakui90** [reply]
yes, you have to subscribe to use service. This Download helper has still downloaded this my using site but after happen failed download, downloaded file  is stuck if I open by VLC player
